libkmod/libkmod-module.c

```diff
@@ -138,17 +137,18 @@ int kmod_module_parse_depline(struct kmod_module *mod, char *line)
 	if (mod->path == NULL) {
 		const char *str = path_join(line, dirnamelen, buf);
 		if (str == NULL)
-			return 0;
+			return;
 		mod->path = strdup(str);
 		if (mod->path == NULL)
-			return 0;
+			return;
```

Comment:
We should probably `free(mod->path); mod->path = NULL;` in the error path.

Just like the others it's pre-existing to feel free to keep it separate patch/PR.